### DISCUSSION TITLE
Limit 0-day hosted server options to 3?

### DISCUSSION THREAD

**Discussion by kraker** [livialima/linuxupskillchallenge#3] [General]
I feel like all of the options for spinning up a server to do the course may be confusing. I wonder if we should just limit the number of options to the free tier on the 3 main CSP's: GCP, Azure, and AWS? This would also streamline maintaining up-to-date docs for all of the solutions. Maybe we can link to 3rd party sources for alternative recommendations at the end of the day-0 article if people want to try something else?

**Comment by livialima** [comment]
I like the idea of using 3rd party sources to limit the number of solutions we actually have a guide to.
The most used (or at least mentioned) options are AWS, DigitalOcean and local servers.
Even with guides for a bunch more, I never really see anyone asking questions about them (Azure and GCP, for example, were never mentioned in the 4 years the challenge has been around).
